docs/pages/components/time-picker.md

```diff
@@ -25,56 +25,48 @@ A basic example of a time picker.
     <div class="fd-popover fd-popover--no-arrow">
         <div class="fd-popover__control">
             <div class="fd-input-group fd-input-group--after">
-                <input type="time" class="fd-input " id="" placeholder="hh:mm am/pm">
+                <input type="time" class="fd-input" id="" placeholder="hh:mm am/pm">
                 <span class="fd-input-group__addon fd-input-group__addon--after fd-input-group__addon--button">
-                                  <button class=" fd-button--light sap-icon--history fd-popover__control" aria-controls="rthHR811" aria-expanded="false" aria-haspopup="true"></button>
+                                  <button class="fd-button--light sap-icon--history fd-popover__control" aria-controls="rthHR811" aria-expanded="false" aria-haspopup="true"></button>
                       </span>
             </div>
         </div>
         <div class="fd-popover__body fd-popover__body--no-arrow" aria-hidden="true" id="rthHR811">
             <div class="fd-time">
                 <div class="fd-time__item">
                     <div class="fd-time__control">
-                        <button class=" fd-button--standard fd-button--light sap-icon--navigation-up-arrow" aria-label="Increase hours" aria-controls="BW7dC141"></button>
-                    </div>
-                    <div class="fd-time__input">
-                        <input class="fd-form__control" type="text" placeholder="hh" value="" id="BW7dC141" aria-label="Hours">
+                        <button class="fd-button--standard fd-button--light sap-icon--navigation-up-arrow" aria-label="Increase hours" aria-controls="BW7dC141"></button>
                     </div>
+                    <input class="fd-time__input fd-form__control" type="text" placeholder="hh" value="" id="BW7dC141" aria-label="Hours">
```

Comment:
These changes—removing wrapping div—were actually `Time` related, but needed to be here too (oops).